lib/mariaex/protocol.ex

```diff
@@ -323,6 +323,19 @@ defmodule Mariaex.Protocol do
     end
   end
 
+  def sanitize_query(query, conn) when is_pid(conn) do
+    {:ok, pool_ref, _mod, state} = DBConnection.Holder.checkout(conn, [])
+    DBConnection.Holder.checkin(pool_ref)
+    sanitize_query(query, state)
+  end
+  def sanitize_query(query, %{opts: opts}) do
+    case Keyword.get(opts, :prepare, :named) do
+      :unnamed -> %Query{query | name: "_unnamed_"}
```

Comment:
This should be `""` not `"__unnamed__"` to match the semantics of postgrex.

Reply:
Using `""` resulted in an increasing amount of prepared statements (in other words: that did not solve the issue).
Using any other string did, so I choose for something describing as `"_unnamed_"`.
But probably we need to alter `prepare_insert` to make `""` work?

Reply:
Done.